modules/frontend/search_handlers.go

```diff
@@ -139,6 +139,8 @@ func logResult(logger log.Logger, tenantID string, durationSeconds float64, req
 	statusCode := -1
 	if httpResp != nil {
 		statusCode = httpResp.StatusCode
+	} else if st, ok := status.FromError(err); ok {
```

Comment:
is httpResp nil in this case?

we may also want to add this same logic to the other log methods